components/gcp/dataflow/launch_python/README.md

```diff
@@ -0,0 +1,116 @@
+
+# Dataflow - Launch Python
+
+Launch a self-executing beam python file. The input python code will be executed inside the KFP's cluster with Dataflow Runner. Client can optionally provide a requirements file to install additional packages to the container. The code will be executed under python 2.7, which is supported beam SDK.
```

Comment:
Recommend replacing the first two lines with "A Kubeflow Pipeline component to submit a Apache Beam job authored in python, to Google Cloud Dataflow for execution".

The second statement can be confusing because folks will perceive that to mean the code is running in the K8s cluster.

Reply:
Done